instrumentation-genai/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/patch.py

```diff
@@ -76,6 +73,12 @@ def traced_method(wrapped, instance, args, kwargs):
                     _set_response_attributes(
                         span, result, event_logger, capture_content
                     )
+                if getattr(result, "choices", None):
+                    for choice in result.choices:
```

Comment:
small nit to cut down on nesting
```suggestion
                for choice in getattr(result, "choices", []):
```